Make the Reviewer's third verdict reachable and drop the Author's status enum (#627)
aunger/gallery-button-for-pixel-camera: agents/dev_orchestration.md

- Dispatch using the dispatch template above, with the Programmer role assignment statement and the literal issue number and branch name tokens.

When the Author returns, apply this transition.

> Apply it to the PR if the Author opened one; apply it to the issue if the Author opened no PR, since that is where the work's labels live when there is no PR:

Author: ```suggestion
Apply it to the PR if one exists; apply it to the issue otherwise, since that is where the work's labels live when there is no PR:
```
It doesn’t matter whether the Author created it. The PR might have pre-existed.

Author: 🤖 Author

Agreed, a PR can pre-exist independently of the Author. Applied your suggestion in 01a28e7.

- Create a sub-agent at the Reviewer model (see Model selection above)
- Dispatch using the dispatch template above, with the Reviewer role assignment statement and the literal issue number token.

> - If the Author opened no PR, the Reviewer examines the Author's explanatory comment on the **issue** rather than a PR.

Author: ```suggestion
- If no PR exists, the Reviewer examines the Author's explanatory comment on the **issue** rather than a PR.
```

Author: 🤖 Author

Applied in 01a28e7.

Monitor output lines are relayed to the user verbatim; this is user-facing status reporting and is not governed by the say-nothing rule (which covers sub-agent messages only).

When the Reviewer returns, apply this transition.

> Apply it to the PR if one was opened; apply it to the issue on the no-PR path, since that is where the work's labels live when there is no PR:

Author: ```suggestion
Apply it to the PR if one exists; apply it to the issue otherwise, since that is where the work's labels live when there is no PR:
```

Author: 🤖 Author

Applied in 01a28e7 for wording consistency with the transition above.

|---|
| `changes requested` |

> Whether a PR was opened in the prior round decides which routing fence applies.

Author: ```suggestion
Whether a PR exists decides which routing fence applies.
```

Author: 🤖 Author

Applied in 01a28e7.

> If no PR was opened, there is no diff or CI to run, so follow the no-PR routing fence; if a PR was opened, follow the Monitor loop fence.

Author: ```suggestion
If no PR exists, there is no diff or CI to run, so follow the no-PR routing fence; if a PR exists, follow the Monitor loop fence.
```

Author: 🤖 Author

Applied in 01a28e7.

Stop the automated cycle and escalate to the User in either of these cases:

> - A sub-agent emits `Cannot work` (see the routing fences above): escalate immediately, without waiting for further rounds.

Author: ```suggestion
- A Reviewer emits `Cannot work` (see the routing fences above): escalate immediately, without waiting for further rounds.
```
The Author can’t do this

Author: 🤖 Author

Correct, the Author cannot emit `Cannot work`; only the Reviewer does. Applied in 06d180b.
